### ISSUE TITLE
Add Mission Statement

### ISSUE THREAD

**Issue by hegemonia** [amaidah/better-vote#37]

**Comment by amaidah** [comment]
hmm, I don't think it should be a card.

Would look better as a full width section in different background color.

**Comment by hegemonia** [comment]
I think that might look a bit... stripey.

**Comment by amaidah** [comment]
just has to be done right

http://foundation.zurb.com/

http://getbootstrap.com/

https://semantic-ui.com/

**Comment by hegemonia** [comment]
If we want something close to that look and feel, that sound like full width white text on the same blue body background.

**Comment by amaidah** [comment]
It sounds a bit wordy.

Can you distill it down to the essence:

Patagonia’s Mission Statement
Build the best product, cause no unnecessary harm, use business to inspire and implement solutions to the environmental crisis.

**Comment by amaidah** [comment]
To support collaborative and democratic processes through the open development and distribution of a secure, flexible, and scalable platform for online voting.
